public/gameInfo.php

```diff
@@ -253,7 +253,41 @@ class='break-words whitespace-normal leading-normal'
 );
 ?>
 <?php if ($isFullyFeaturedGame): ?>
```

Comment:
Not sure if you want to tackle it as part of this PR, but the mature hub warning doesn't go through this path, and that's what I'd expect web crawlers to encounter:
```
<meta property="og:title" content="~Homebrew~ Larry and the Long Look for a Luscious Lover (NES)">
<meta property="og:description" content="~Homebrew~ Larry and the Long Look for a Luscious Lover (NES)">
<meta property="og:image" content="http://localhost/assets/images/favicon.webp">
<meta property="og:url" content="http://localhost/game/17703">
<meta property="og:type" content="website">
```
After warning:
```
<meta property="og:title" content="~Homebrew~ Larry and the Long Look for a Luscious Lover (NES)">
<meta property="og:description" content="There are 54 worth 275 (299) points. ~Homebrew~ Larry and the Long Look for a Luscious Lover for NES - explore and compete on this classic game at RetroAchievements.">
<meta property="og:image" content="http://media.retroachievements.org/Images/047004.png">
<meta property="og:url" content="http://localhost/game/17703">
<meta property="og:type" content="retroachievements:game">
```

Reply:
Similarly, hubs aren't handled.
```
<meta property="og:title" content="[Series - Dragon Quest \| Dragon Warrior] (Hubs)">
<meta property="og:description" content="[Series - Dragon Quest \| Dragon Warrior] (Hubs)">
<meta property="og:image" content="http://localhost/assets/images/favicon.webp">
<meta property="og:url" content="http://localhost/game/6708">
<meta property="og:type" content="website">
```

Reply:
Probably worth handling these as a separate unit of work. Note: Because the title and description are the same, Google is falling back to using another tag we render into `<head>`:

```html
<meta name="description" content="Adding achievements to your favourite retro games since 2012">
```

For the mature hub games, I think this is probably okay:
![Screenshot 2023-07-11 at 5 04 22 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/2e608d0d-eeac-4d18-a98a-6249c87d27ba)

---

For hubs themselves, the same thing is happening, but we can probably do a lot to enhance this experience:
![Screenshot 2023-07-11 at 5 47 37 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/11f87d94-cbb1-4b56-90e0-8dc48ca43c6d)

However, I must confess I don't want to add more "special" hub logic into _gameInfo.php_ unless absolutely necessary.